src/crystal/system/system.cr

```diff
@@ -0,0 +1,24 @@
+# :nodoc
+module Crystal
+  # :nodoc
+  module System
+    # :nodoc
+    module System
```

Comment:
no but place the methods in `Crystal::System` not `Crystal::System::System`. It seems redundant and these are "misc" methods which can just go in `Crystal::System` itself.